server/enforcer.go

```diff
@@ -30,6 +31,7 @@ import (
 type Server struct {
 	enforcerMap map[int]*casbin.Enforcer
 	adapterMap  map[int]persist.Adapter
+	mu          sync.RWMutex
```

Comment:
Just one comment- as there are 2 maps better to use 2 mutexes (else there will be a scenario where you will be blocking write to one map when writing to the other map).

Reply:
😰you are right. My baaaad